index.d.ts

```diff
@@ -0,0 +1,11 @@
+import * as _install from './install';
+import * as preprocessor from './preprocessor';
+
+interface TsJestModule {
+  install: typeof _install.install;
+  process: typeof preprocessor.process;
+  getCacheKey: typeof preprocessor.getCacheKey;
+}
+
+declare const tsJestModule: TsJestModule;
+export = tsJestModule;
```

Comment:
@kulshekhar:
While writing the new `ts-jest-module-interface` test case, I found out that `dist/index.d.ts` was missing.
I just added this file to replace it. This reflects the manually written `index.js`. Its gets copied to `dist/` as part of the `"build"` package script.